pkg/armhelpers/httpMockClient.go

```diff
@@ -19,7 +19,7 @@ const (
 	tenantID                                   = "19590a3f-b1af-4e6b-8f63-f917cbf40711"
 	resourceGroup                              = "TestResourceGroup"
 	computeAPIVersion                          = "2019-07-01"
-	diskAPIVersion                             = "2019-07-01"
+	diskAPIVersion                             = "2019-11-01"
```

Comment:
Is this related to the compute API bump?

Reply:
correct, the disk api version is already `2019-11-01` in v40.2.0, need to change the version in mock client, otherwise unit test failed.